frontend/pages/SoftwarePage/SoftwareVulnerabilities/SoftwareVulnerabilitiesTable/helpers.ts

```diff
@@ -5,18 +5,18 @@ export const getExploitedVulnerabilitiesDropdownOptions = (
 
   return [
     {
-      disabled: false,
+      isDisabled: false,
       label: "All vulnerabilities",
-      value: false,
+      value: "false",
       helpText: "All vulnerabilities detected on your hosts.",
     },
     {
-      disabled: !isPremiumTier,
+      isDisabled: !isPremiumTier,
       label: "Exploited vulnerabilities",
-      value: true,
+      value: "true",
       helpText:
         "Vulnerabilities that have been actively exploited in the wild.",
-      tooltipContent: !isPremiumTier && disabledTooltipContent,
+      tooltipContent: !isPremiumTier ? disabledTooltipContent : undefined,
```

Comment:
Same Q here of swithing from `&&` to the-construct-golang-is-missing

Reply:
fixed! ty

Reply:
Ahh, actually, for this one it was intentional, because `!isPremiumTier && disabledTooltipContent` can evaluate to `false` which tooltipContent type is only type `string` or `undefined`.

I totally forgot, but remembered now that I put it back and it's breaking the build (see screenshot).
<img width="1057" alt="Screenshot 2025-01-09 at 12 03 11 PM" src="https://github.com/user-attachments/assets/49630a39-48be-4f02-a836-870c408a2bfb" />